test/infrastructure/docker/internal/docker/machine.go

```diff
@@ -340,7 +365,7 @@ func (m *Machine) ExecBootstrap(ctx context.Context, data string, format bootstr
 
 	switch format {
 	case bootstrapv1.CloudConfig:
-		commands, err = cloudinit.RawCloudInitToProvisioningCommands(cloudConfig)
+		commands, err = cloudinit.RawCloudInitToProvisioningCommands(cloudConfig, kindMapping)
```

Comment:
What is the impact of implementing this only for cloudinit. Does this mean in our ignition test we end up with a different config?

Reply:
I'm going to open an issue for ignition, but I don't have enough knowledge/bandwidth to take care of it.
Also, we are not using ignition as extensively as cloud init, so I hope there will be less problems there

Reply:
I would assume that as soon as we drop something relevant from our cluster-templates the ignition quickstart will fail.

The ignition cluster-template today uses the same base YAMLs as the other templates.

I'm fine with a follow-up. But I would imagine that we either have to use different kubletExtraArgs for the ignition cluster-template or we have to implement it for ignition as well.
(when you open an issue, please also add that the `ignore-preflight-errors` stuff is already out of sync between cloud-init and ignition and this should be fixed as well)